fincm3.coffee

```diff
@@ -0,0 +1,50 @@
+deviceTypesCommon = require '@resin.io/device-types/common'
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+
+FIN_DEBUG = "While not having the Fin board powered, connect your system to the board's DBG port via a micro-USB cable.."
+FIN_POWER = "Power on the Fin by attaching power to either the Barrel or the Phoenix connector."
+FIN_WRITE = "Write the OS to the internal MMC storage device. We recommend using <a href=http://www.etcher.io/>Etcher</a>."
+FIN_POWEROFF = "When flashing is complete, power off the board by detaching the power and unplug the DGB micro-USB cable."
+
+module.exports =
+	version: 1
+	slug: 'fincm3'
+	aliases: [ 'fincm3' ]
+	name: 'Balena Fin (CM3)'
+	arch: 'armv7hf'
+	state: 'released'
+
+	subtype_of: 'raspberrypi3'
```

Comment:
Would there ever be a case where we would want this to be an array? if the device type was a sub type of 2 or more other types?

Reply:
Don't see that as a possibility. What do you think, @petrosagg ?

Reply:
@shaunmulligan Shall we merge it like this for now and figure it out if needed change in a later iteration?